index.html

```diff
@@ -34,65 +34,215 @@ <h3 class="masthead-brand">resPOSTa!</h3>
+                            ng-click="join()"
+                            >Join</a>
+                        </form>
+                    </p>
+                    <div class="alert alert-warning" role="alert" ng-show="form.input.$error.required">
+                        Room name is required!
+                    </div>
+                    <div class="alert alert-warning" role="alert" ng-show="form.input.$error.pattern">
+                        Room name must be a single English word!
+                    </div>
+                    <button type="button" class="btn btn-danger btn-lg" data-toggle="modal" data-target="#newRmDialog">CREATE A ROOM</button>
 
-						<div class="alert alert-warning" role="alert" ng-show="form.input.$error.required">
-							Room name is required!</div>
-							<div class="alert alert-warning" role="alert" ng-show="form.input.$error.pattern">
-								Room name must be a single English word!
-							</div>
+                    <!-- Modal -->
+                    <div class="modal fade" id="newRmDialog" role="dialog">
+                        <div class="modal-dialog">
 
-						</div>
-
-						<div class="mastfoot">
-							<div class="inner">
-								<p>&copy;<a href="https://github.com/CodeMonkeys3111">CodeMonkeys</a> 2015, All Rights Reserved.</p>
-								<p>Powered by <a href="http://getbootstrap.com">Bootstrap</a>, <a href="https://twitter.com/mdo">@mdo</a>.</p>
-							</div>
-						</div>
-
-					</div>
+                            <!-- Modal content-->
+                            <div id="newRoom" class="modal-content">
+                                <div class="modal-header">
+                                    <button type="button" class="close" data-dismiss="modal">&times;</button>
+                                    <h4 class="modal-title"><span class="glyphicon glyphicon-pushpin"></span> Create A New Room</h4>
+                                </div>
+                                <div class="modal-body">
+                                    <form role="form" name="CreateRoom">
+                                        <div class="form-group">
+                                            <label for="usrname"><span class="glyphicon glyphicon-home"></span> Room Name</label>
+                                            <input type="text" class="form-control" name="usrname" placeholder="Your Room Name"
+                                                   ng-model="newRoomName.text" ng-pattern="newRoomName.word" required>
+                                            <p role="alert" ng-show="CreateRoom.usrname.$error.required">
+                                                <b style="color: red;">Room name is required!</b>
+                                            </p>
+                                            <p role="alert" ng-show="CreateRoom.usrname.$error.pattern">
+                                                <b style="color: orange;">Room name must be English letters!</b>
```

Comment:
"Room name must only contain English letters and numbers. "
or 
"Room name must only contain alphanumeric characters."

Fix this.